servers/akumamc/metadata.json

```diff
@@ -1,9 +1,11 @@
 {
     "id": "akumamc",
     "name": "AkumaMC",
+    "description" : "A Java and Bedrock compatible server with OP Prison and Skyblock gamemodes",
     "website": "https://akumamc.net/",
     "store": "https://store.akumamc.net/",
     "addresses": [
+        "lunar.akumamc.net",
```

Comment:
This part isn’t needed, we use a suffix-based matching system so anything under akumamc.net will handle it

Reply:
I assumed this was the case, but was asked to add it explicitly - I've got ahead and removed it.